nodewatcher/modules/monitor/http/survey/processors.py

```diff
@@ -0,0 +1,92 @@
+import datetime
+import pytz
+
+from django.utils.translation import gettext_noop
+
+from nodewatcher.core.monitor import models as monitor_models, processors as monitor_processors
+from nodewatcher.utils import ipaddr
+from nodewatcher.modules.monitor.sources.http import processors as http_processors
+
+from django_datastream import datastream
+
+DATASTREAM_SUPPORTED = False
+try:
+    from nodewatcher.modules.monitor.datastream import fields as ds_fields, models as ds_models, base as ds_base
+    from nodewatcher.modules.monitor.datastream.pool import pool as ds_pool
+
+    class SurveyInfoStreams(ds_base.StreamsBase):
+        neighbor_graph = ds_fields.GraphField(tags={
+            'title': gettext_noop("Neighbor topology"),
+            'description': gettext_noop("Neighbor topology."),
+            'visualization': {
+                'type': 'graph',
+                'initial_set': True,
+            }
+        })
+
+        def get_module_name(self):
+            return 'monitor.http.survey'
+
+        def get_stream_query_tags(self):
+            return {'module': 'survey'}
+
+        def get_stream_tags(self):
+            return {'module': 'survey'}
+
+        def get_stream_highest_granularity(self):
+            return datastream.Granularity.Minutes
```

Comment:
This is done by [nodewatcher agent](https://github.com/wlanslovenija/nodewatcher-agent). See here: https://github.com/wlanslovenija/nodewatcher-agent/blob/master/modules/wireless.c#L362